Jenkinsfile

```diff
@@ -52,8 +52,8 @@ pipeline {
               sh """
                 echo 'Build Helm chart'
                 helm init --client-only --stable-repo-url=https://charts.helm.sh/stable
-                helm repo add kubernetes-charts https://kubernetes-charts.storage.googleapis.com/
-                helm repo add kubernetes-charts-incubator http://storage.googleapis.com/kubernetes-charts-incubator
+                helm repo add stable-charts https://charts.helm.sh/stable
```

Comment:
I think this one is not strictly needed thanks to `helm init --client-only --stable-repo-url=https://charts.helm.sh/stable`.